nipype/interfaces/base.py

```diff
@@ -1148,6 +1155,34 @@ def version(self):
                                  self.__class__.__name__)
         return self._version
 
+    def load_inputs_from_json(self, json_file, overwrite=True):
+        """
+        A convenient way to load pre-set inputs from a JSON file.
+        """
+
+        with open(json_file) as fhandle:
+            inputs_dict = json.load(fhandle)
```

Comment:
mmm, I'm getting object does not have update. (?)

Anyways, I will use sets and get_traitsfree to implement this in a cleaner manner.